### ISSUE TITLE
Add articles for Arnoldia, 0004-2633

### ISSUE THREAD

**Issue by suwiding** [rdmpage/biostor#44]
@rdmpage This publication will be scanned and added to BHL shortly.  I  received a full set of article metadata for the publication from the Arnold Arboretum and will provide 'scrubbed' article metadata after the content is uploaded.  Is this content within scope for BioStor? Can we handle these articles the same way that we handled the articles for other publications, e.g. Phasmid Studies?

**Comment by rdmpage** [comment]
Yes, absolutely! From my perspective any journal scanned by BHL is automatically within scope for BioStor.

**Comment by trosesandler** [comment]
Great.  Who is doing the scanning?  Anything i need to do on my end?
